web-admin-app/src/main/java/edu/unc/lib/boxc/web/admin/controllers/ChompbController.java

```diff
@@ -0,0 +1,16 @@
+package edu.unc.lib.boxc.web.admin.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+/**
+ * @author lfarrell
+ */
+@Controller
+public class ChompbController {
```

Comment:
Should add a really basic test for this. Shouldn't need any spring context files, see tests like DownloadImageControllerIT for an example, can just use
```
mvc = MockMvcBuilders.standaloneSetup(controller).build();
```